install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -741,10 +741,12 @@ label.
 
 This means two things for unprivileged containers:
 
-- The volume will be given a `type` which is accessible by unprivileged
-containers. This type is usually `container_file_t`, which treats volumes as container content. Previously, the label specified was `svirt_sandbox_file_t`. This label
-is no longer used due to security concerns.
-- If a `level` is specified, the volume will be labeled with the given MCS
+- The volume is given a type that is accessible by unprivileged containers.
+This type is usually `container_file_t` in RHEL 7.5+, which treats volumes as
```

Comment:
Since you should spell out RHEL on first use, I'd say "Red Hat Enterprise Linux (RHEL) version 7.5 and later" :woman_shrugging: